cypress/e2e/MachineActions.cy.ts

```diff
@@ -0,0 +1,28 @@
+describe("Machine Actions Integration Test", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+
+  it("should add and delete machine actions in the gueue", () => {
+    cy.get('[data-testId="action-input"]').type("Action 1");

```

Comment:
In cypress you can create your own selectors to avoid typing attribute full name